workers/unity/Packages/com.improbable.gdk.deploymentlauncher/TaskManager.cs

```diff
@@ -0,0 +1,121 @@
+using System;
+using System.Collections.Generic;
+using Improbable.Gdk.DeploymentManager.Commands;
+using UnityEditor;
+
+namespace Improbable.Gdk.DeploymentManager
+{
+    internal class TaskManager
+    {
+        public enum QueueMode
+        {
+            Normal,
+            ForceNext
```

Comment:
`Enqueue`, `RunNext`?

Reply:
`PushFront` `PushBack` ? (Stealing from the C++ deque implementation)